ff-netF48-server-sdk.csproj

```diff
@@ -5,9 +5,9 @@
     <PackageId>ff-dotnet-server-sdk</PackageId>
     <RootNamespace>io.harness.cfsdk</RootNamespace>
     <GeneratePackageOnBuild>false</GeneratePackageOnBuild>
-    <Version>1.1.5</Version>
+    <Version>1.1.6</Version>
     <PackOnBuild>true</PackOnBuild>
-    <PackageVersion>1.1.5</PackageVersion>
+    <PackageVersion>1.1.6</PackageVersion>
     <Authors>andrija.milovanovic@harness.io,milos.vasic@harness.io</Authors>
```

Comment:
we should probably change the emails here to support@harness.io at some point

Reply:
Done, I've also removed Owners which resolved to a broken URL